scrapi/harvesters/umontreal.py

```diff
@@ -7,14 +7,31 @@
 '''
 from __future__ import unicode_literals
 
+from scrapi.base.helpers import updated_schema
 from scrapi.base import OAIHarvester
 
 
+def umontreal_language_processor(languages):
+
+    if not languages:
+        languages = []
+
+    return languages
```

Comment:
Below is the description in the schema:
- "description": "The primary languages in which the content of the resource is presented. Values used for this element MUST conform to ISO 639\u20133. This offers two and three letter tags e.g. \"en\" or \"eng\" for English and \"en-GB\" for English used in the UK."

You can find all the tags here: https://www.loc.gov/standards/iso639-2/php/code_list.php

It is my understanding that the tags are correct - en maps to English and fr maps to French

Can you confirm that this still needs to be done?